tests/test_bukuDb.py

```diff
@@ -13,14 +13,13 @@
 from random import shuffle
 from unittest import mock
 
-from genericpath import exists
```

Comment:
This line was somehow re-added in the last merge to `master` (resulting in a linting failure) :sweat_smile: